lists/kh/kh-ingo.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": {
+    "en": "Ministry of Foreign Affairs",
+    "local": ""
+  },
+  "url": "https://www.mfaic.gov.kh/",
+  "description": {
+    "en": "A foreign NGO must register with the MOFA by entering into a Memorandum of Understanding (“MOU”) with MOFA. After the MOFA has entered into the MOU with a foreign NGO, the NGO must sign a Project Agreement with the MOFA before starting its work.  A foreign NGO may be established for a three (3) year duration which must be renewed with the MOFA before the expiration of the term.\nAn organization registered as a foreign NGO by the MOFA is considered properly licensed to carry out its activities in Cambodia and no further registration is required. However, in the event a foreign organization was to carry out business activities pursuant to Article 272 of the LOCE it may remain subject to registration as in accordance with that provision.\nAfter obtaining MOFA or MOI approval to be established, a foreign or local NGO must also register with the CDC if it wishes to import duty free materials to serve its program. The CDC will periodically require the NGO complete several forms."
```

Comment:
It looks to me like there is no registration number or similar which can actually be used as an ID for NGOs registered under an MOU. Is that right? If so, I suggest noting that explicitly here under the description. (Or do you have some information via the IATI helpdesk which can be used to create an ID?)

Reply:
Correct - there is no number just a MOU - I can provide an example of what that document looks like?